containers/ecr-viewer/src/app/api/services/listEcrDataService.ts

```diff
@@ -149,52 +81,10 @@ export const processListPostgres = (responseBody: any[]): EcrDisplay[] => {
 };
 
 /**
- * Processes a list of eCR data retrieved from an S3 bucket.
- * @param responseBody - The response body containing eCR data from S3.
- * @returns - The processed list of eCR IDs and dates.
+ * Retrieves the total number of eCRs stored in the fhir table.
+ * @returns A promise resolving to the total number of eCRs.
  */
-export const processListS3 = (
-  responseBody: ListObjectsV2CommandOutput,
-): any[] => {
-  responseBody.Contents?.sort(
-    (a, b) => Number(b.LastModified) - Number(a.LastModified),
-  );
-
-  return (
-    responseBody.Contents?.map((object) => {
-      return {
-        ecrId: object.Key?.replace(".json", "") || "",
-        dateModified: object.LastModified
-          ? convertUTCToLocalString(
-              formatDateTime(new Date(object.LastModified!).toISOString()),
-            )
-          : "",
-      };
-    }) || []
-  );
-};
-
-/**
- * Fetches eCR Metadata stored in the fhir_metadata table.
- * @param processedData - List of eCR IDs.
- * @returns ecr metadata
- */
-const getFhirMetadata = async (
-  processedData: EcrDisplay[],
-): Promise<EcrMetadataModel[]> => {
-  const isNonIntegratedViewer =
-    process.env.NEXT_PUBLIC_NON_INTEGRATED_VIEWER === "true";
-  if (isNonIntegratedViewer) {
-    const ecrIds = processedData.map((ecr) => ecr.ecrId);
-    const fhirMetadataQuery =
-      "SELECT ecr_id, patient_name_last, patient_name_last, patient_birth_date, report_date, reportable_condition, rule_summary FROM fhir_metadata where ecr_id = $1";
-
-    const data = await database.manyOrNone<EcrMetadataModel>(
-      fhirMetadataQuery,
-      [ecrIds],
-    );
-    return data;
-  } else {
-    return [];
-  }
+export const getTotalEcrCount = async (): Promise<number> => {
+  let number = await database.manyOrNone("SELECT count(*) FROM fhir_metadata");
```

Comment:
sorry, didn't see this earlier, but I think this can just be `one` instead of `manyOrNone` since count should only ever return 1 value.
```suggestion
  let number = await database.one("SELECT count(*) FROM fhir_metadata");
```